spark/v3.5/spark-runtime/LICENSE

```diff
@@ -582,66 +460,48 @@ License text:
 --------------------------------------------------------------------------------
 
 This binary artifact contains code from Project Nessie.
-
-Copyright: 2020 Dremio Corporation.
+Group: org.projectnessie.nessie Name: nessie-client Version: 0.102.2
+Copyright: 2015-2017 Dremio Corporation
 Home page: https://projectnessie.org/
 License: http://www.apache.org/licenses/LICENSE-2.0
 
 --------------------------------------------------------------------------------
 
-This product includes code from Apache Spark.
-
-* vectorized reading of definition levels in BaseVectorizedParquetValuesReader.java
-* portions of the extensions parser
-* casting logic in AssignmentAlignmentSupport
-* implementation of SetAccumulator.
-
+Group: org.apache.spark Name: spark-sql_2.12 Version: 3.5.4
 Copyright: 2011-2018 The Apache Software Foundation
 Home page: https://spark.apache.org/
 License: https://www.apache.org/licenses/LICENSE-2.0
 
 --------------------------------------------------------------------------------
 
-This product includes code from Delta Lake.
-
-* AssignmentAlignmentSupport is an independent development but UpdateExpressionsSupport in Delta was used as a reference.
-
-Copyright: 2020 The Delta Lake Project Authors.
-Home page: https://delta.io/
-License: https://www.apache.org/licenses/LICENSE-2.0
+Group: org.apache.httpcomponents Name: httpclient Version: 4.5.14
+Copyright: 1999-2021 The Apache Software Foundation
+Home page: https://hc.apache.org/
+License: http://www.apache.org/licenses/LICENSE-2.0
 
 --------------------------------------------------------------------------------
 
-This binary includes code from Apache Commons.
-
-* Core ArrayUtil.
-
-Copyright: 2020 The Apache Software Foundation
-Home page: https://commons.apache.org/
-License: https://www.apache.org/licenses/LICENSE-2.0
+Group: dev.failsafe Name: failsafe Version: 3.3.2
```

Comment:
Can you please roll back the reformatting? It makes this much harder to review.

Reply:
Yes, I agree.